packages/discord.js/src/structures/MessageEmbed.js

```diff
@@ -88,8 +88,8 @@ class MessageEmbed {
      * @type {?number}
      */
     // Date.parse() cannot be used here because data.timestamp might be a number
-    // Additionally, the nullish coalescing operator cannot be used here as we're checking for 0
-    this.timestamp = new Date(data.timestamp).getTime() || null;
+    // eslint-disable-next-line eqeqeq
+    this.timestamp = data.timestamp != null ? new Date(data.timestamp).getTime() : null;
```

Comment:
I'm just curious to know that why we can't use `!==` here?

Reply:
Read the description

Reply:
Oh sorry, got it's because of 0. right?

Reply:
No... it's because of undefined